### DISCUSSION TITLE
[Regression] why was transmit_power setting removed in 1.17.1-1 ?

### DISCUSSION THREAD

**Discussion by thiscantbeserious** [Koenkk/zigbee2mqtt#6251] [General]
I'm using zigbee2mqtt in Home Assistant as an Addon and in 1.17.1-1 stable you've specifically removed this option only from the settings - the code itself is still there, just non-functional.

Why? Was it ment to be only for dev - or did you plan to move it to another config sub-section (e.g. advanced), just forgot to do it?

**Comment by gnn3820** [comment]


**Reply by timoline** [reply]
I know its just an example, but please not in advanced section, but in a section where it belongs to :-)

**Reply by gnn3820** [reply]
wherever you like, I thought it was a general discussion, you mean comments specific to the regression?

**Comment by Koenkk** [comment]
Nothing has changed regarding this option from the z2m perspective. Likely something in the addon so create an issue her: https://github.com/danielwelch/hassio-zigbee2mqtt/issues

**Reply by thiscantbeserious** [reply]
Alright, thanks ... did that. :)

https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/29
